Psionic Content

ZeroDayDaemon opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Adds a few pieces of psionic clothing
:cl: Daemon
- add: Added new psionic clothing.

ZeroDayDaemon committed:
Psionic Content

Elijahrane commented on Content.Server/Nyanotrasen/Abilities/Psionics/Abilities/NoosphericZapPowerSystem.cs:
```diff
@@ -0,0 +1,64 @@
+
+        private void OnPowerUsed(NoosphericZapPowerActionEvent args)
+        {
+            if (!(TryComp<DamageableComponent>(args.Target, out var damageable) && damageable.DamageContainerID == "Biological"))
```
your test component should be if they have PotentialPsionicComponent

Elijahrane commented on Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml:
```diff
@@ -0,0 +1,88 @@
+  components:
+  - type: Headset
+    channels:
+    - Telepathic
```
telepathic isn't a channel

ZeroDayDaemon committed:
Address Reviews

Elijahrane commented on Resources/Prototypes/Nyanotrasen/Entities/Clothing/PsionicClothing.yml:
```diff
@@ -0,0 +1,89 @@
+    sprite: Clothing/Ears/Headsets/syndicate.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/syndicate.rsi
+#Functionality not coded yet
```
dont really want placeholders for these

Elijahrane replied:
just delete the headsets

ZeroDayDaemon committed:
rane bitches too much

ZeroDayDaemon committed:
Tonight, on unsolved mysteries, find out who gives a shit about bigfoot

UPDATE: apparently nobody gives a shit so fuck him.

UPDATE: Last night somebody broke in and stole $500 worth of shit at my house

That’s right $500 worth of BULLSHIT!!!

ZeroDayDaemon committed:
Address reviews
